man/dockerd.8.md

```diff
@@ -441,7 +441,7 @@ device. And devices automatically go away when last user of the device
 exits.
 
 For example, when a container exits, its associated thin device is removed. If
-that device has leaked into some other mount namespace and can't be removed,
+that device has leaked into some other mount namespaces and can't be removed,
```

Comment:
Thought for a while,i agree you point,thanks.@thaJeztah 
And @cpuguy83's answer ,it may well be a good idea,what do you think about this? @thaJeztah